collections/Douce/MS_Douce_367.xml

```diff
@@ -37,45 +40,121 @@
                      <idno type="SCN">21942</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Johannes Platterberger and Dietrich Truchseß, <title>Excerpta chronicarum</title>, part I; Germany (?Nuremberg), 15th century, second half (after 1459)</head>
                <msContents>
                   <msItem n="1" xml:id="MS_Douce_367-item1">
                      <author key="person_5334019">Johannes Platterberger</author>
                      <author key="person_57047805">Dietrich Truchseß</author>
                      <title key="work_2514">Excerpta chronicarum</title>
                      <note>(universal chronicle; part i)</note>
+                     <msItem>
+                        <locus>(fols. 1r–6r)</locus>
+                        <note>Table of chapters</note>
```

Comment:
(A thought for some day – we should have a standard way of tagging/finding features such a capitula list or index.)